crates/turborepo-lib/src/process/mod.rs

```diff
@@ -65,7 +65,8 @@ impl ProcessManager {
         if lock.is_closing {
             return None;
         }
-        let child = child::Child::spawn(command, child::ShutdownStyle::Graceful(stop_timeout));
+        let child =
+            child::Child::spawn(command, child::ShutdownStyle::Graceful(stop_timeout), false);
```

Comment:
note to self: 3rd argument is `use_pty` and its hardcoded to false at the moment